web/src/components/repo/settings/RegistriesTab.vue

```diff
@@ -35,6 +35,7 @@
     <div v-else class="space-y-4">
       <form @submit.prevent="createRegistry">
         <InputField label="Address">
+          <!-- TODO: check input field Address is a valid address -->
```

Comment:
@anbraten do you have a nice way to do this in frontend?

Reply:
We can add something like [vuelidate](https://vuelidate-next.netlify.app/) in the future. For now it should show you the error from the backend as a toast notification.

Reply:
ok I'll leave the TODO and we can do this later -> #533